Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -136,15 +144,30 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
     }
 
     private func getEditorConfiguration() -> WKUserScript {
-        let jsCode = """
+        let escapedTitle = initialTitle.addingPercentEncoding(withAllowedCharacters: .alphanumerics)!
+        let escapedContent = _initialRawContent.addingPercentEncoding(withAllowedCharacters: .alphanumerics)!
+        let postId = id != nil ? id! : nil
+        let hasThemeStylesEnabled = themeStyles != nil ? themeStyles! : false
```

Comment:
I believe this might be simplified. WDYT?  
```suggestion
        let postId = id ?? nil
        let hasThemeStylesEnabled = themeStyles ?? false
```

Reply:
Actually, we likely do not even need `let postId = id ?? nil`. We could just pass `id`.

Reply:
I've updated it in this commit https://github.com/wordpress-mobile/GutenbergKit/pull/21/commits/54aedfb4541fb8465a6b4caa2c2c58f1d86334a3

Reply:
Noting that we could still simplify `hasThemeStylesEnabled`, if desired.

Reply:
Oh I missed that, I'll test that suggestion. I was trying to avoid setting a value wrapped with `Optional()`. I'll check if that handles it.

Reply:
Gotcha. It is very possible my suggestion is invalid. Feel free to disregard.